src/js/common/helpers/login-helpers.js

```diff
@@ -39,6 +39,8 @@ export function getUserData(dispatch) {
       dispatch(updateProfileField('status', json.data.attributes.va_profile.status));
       dispatch(updateProfileField('services', json.data.attributes.services));
       dispatch(updateLoggedInStatus(true));
+    } else {
+      dispatch(profileLoadingFinished());
```

Comment:
@raquelromano-gov So, after thinking about it, I think that's probably clearer for the code that's here now.

That said: I mentioned this to Chris the other day: we often dispatch actions in Redux that are very granular and focused on updating one piece of state, but I think it's better to have actions that describe an actual event in the system. Ideally, I think the profile here code would dispatch actions like FETCH_PROFILE, UPDATE_PROFILE_INFO, and PROFILE_FETCH_FAILED (or something like that). The logged in and pending status would be set in actions like that (along with other info) and not have their own specific actions. That way, if you look at a list of actions that have occurred, it's a (somewhat) meaningful history of what your application is doing.